### ISSUE TITLE
cannot see any of the docker containers installed by Deplorarr

### ISSUE THREAD

**Issue by moltra** [lllllllillllllillll/DweebUI#109]
I have 55 containers and over 192 images, but I am not seeing anyone of the ones installed.  I can see where i can install them, but not the existing running containers.

I am using Proxy Socker, so that might be the problem, does this support Proxy Socket?

**Comment by lllllllillllllillll** [comment]
Right now the app is configured to connect to the Docker socket: '/var/run/docker.sock'. 
With the socket proxy, you would be connecting with something like 'tcp://localhost:2375'?

**Comment by lllllllillllllillll** [comment]
@moltra Could you give me some more information about  the Proxy Socket or a link to the project?

**Comment by SimpleHomelab** [comment]
@moltra  Deployarr uses Socket-Proxy. 

@lllllllillllllillll I came to post a similar issue/request - support for Docker Socket Proxy. I am the author of [Deployarr](https://github.com/anandslab/deployarr). DweebUI looks awesome. I see you have a patreon page, do you have an opencollective page? I would love to sponsor the project.

**Comment by moltra** [comment]
@anandslab yes, I know I was at one time very active on your discord.  I still review it at time to time.  I will probably get active on your board once I get my server back up.

**Comment by lllllllillllllillll** [comment]
@anandslab That's an interesting project. My next release won't have the docker.socket requirement, so it shouldn't have any issue with socket-proxy. 
I'll have to take a look at creating an Open Collective page.

**Comment by SimpleHomelab** [comment]
Awesome, i already added DweebUI to the next Deployarr release, with docker socket. I will update it once you add socket proxy support. 

Opening an opencollective page is very easy. Be sure to add a separate organization page for DweebUI so you can keep it separate from from your individual account. I had many issues with Patreon. Plus it appears they take a higher commission.
